src/main/java/dan200/computercraft/shared/peripheral/monitor/ClientMonitor.java

```diff
@@ -68,13 +70,11 @@ public boolean createBuffer( MonitorRenderer renderer )
 
                 tboBuffer = OpenGlHelper.glGenBuffers();
                 OpenGlHelper.glBindBuffer( GL31.GL_TEXTURE_BUFFER, tboBuffer );
-                GL15.glBufferData( GL31.GL_TEXTURE_BUFFER, 0, GL15.GL_STATIC_DRAW );
+                monitorBuffer = null;
```

Comment:
Maybe move this to `deleteBuffers`? It feels a little odd to have it kicking around here.

Reply:
This line doesn't actually do anything, don't know why it's here.

Reply:
Well, I guess there's no harm in having it, as it frees a (tiny) bit of memory. But yes, feels a little odd.

Reply:
Not even that, `monitorBuffer` is `null` to begin with. I think I wrote that down to clarify that it is supposed to start off empty? Either way, I removed it in the latest commit.